sigs.yaml

```diff
@@ -722,6 +722,9 @@ sigs:
     description: Centralized plugin index for krew.
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/krew-index/master/OWNERS
+  - name: krm-functions
+    owners:
+    - https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/OWNERS
```

Comment:
Using OWNERS file of the kustomize project, since we don't have a repo for the subproject yet and the validation requires this field to be a githubusercontent URL.

Reply:
I would hold off the addition of the subproject till the separate repo is created. 
Please open a request here: https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-repo&template=repo-create.yml&title=REQUEST%3A+%3CCreate+or+Migrate%3E+%3Cgithub+repo%3E

Reply:
Thanks for your advice!
I have just created https://github.com/kubernetes/org/issues/3117.